hnn_core/dipole.py

```diff
@@ -81,8 +81,10 @@ def prsimtime():
             cvode.event(tt, prsimtime)  # print time callbacks
 
     h.fcurrent()
-    # set state variables if they have been changed since h.finitialize
-    h.frecord_init()
```

Comment:
testing has shown this is unnecessary (state variables have not changed since line 73)